types/pino/index.d.ts

```diff
@@ -444,6 +444,18 @@ declare namespace P {
          * key-value object added as child logger to each log line. If set to null the base child logger is not added
          */
         base?: { [key: string]: any } | null;
+
+        /**
+         * An object containing functions for formatting the shape of the log lines.
+         * These functions should return a JSONifiable object and should never throw.
+         * These functions allow for full customization of the resulting log lines.
+         * For example, they can be used to change the level key name or to enrich the default metadata.
+         */
+        formatters?: {
+          level?: (level: string, number: number) => object;
+          bindings?: (bindings: Bindings) => object;
+          log?: (object: object) => object;
```

Comment:
Can we add documentation comments to these functions?

I already did the work [here](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/43508#discussion_r404133823), so I copied it over for you. The indentation might need some tweaks.
```suggestion
          /**
           * Changes the shape of the log level. The default shape is `{ level: number }`.
           * @param label The label of the level (e.g. `info`).
           * @param number The numeric value (e.g. `30`).
           * @example
           * level (label, number) {
           *  return { level: number }
           * }
           */
          level?: (level: string, number: number) => object;
          /**
           * Changes the shape of the bindings. The default shape is `{ pid, hostname }`. It will be called every time
           * a child logger is created.
           * @param bindings The bindings object.
           * @example
           * bindings (bindings) {
           *  return { pid: bindings.pid, hostname: bindings.hostname }
           * }
           */
          bindings?: (bindings: Bindings) => object;
          /**
           * Changes the shape of the log object. This function will be called every time one of the log methods (such
           * as `.info`) is called. All arguments passed to the log method, except the message, will be pass to this
           * function. By default it does not change the shape of the log object.
           * @example
           * log (object) {
           *  return object
           * }
           */
          log?: (object: object) => object;
```

Reply:
Thanks @AdamVig , done.